tests/raw_query/models.py

```diff
@@ -23,6 +23,10 @@ class Book(models.Model):
     opening_line = models.TextField()
 
 
+class BookInfo(models.Model):
+    book = models.ForeignKey(Book, primary_key=True, db_column="book")
```

Comment:
`db_column` does need to be specified. The default column name would be "book_id" otherwise (which would *not* trigger the bug).